docs/cloud/earthly-ci.md

```diff
@@ -0,0 +1,149 @@
+* **Remote build runners** - Earthly CI comes with access to Earthly Satellites. This means that you can run builds from your laptop without having to commit code to Git with every attempt.
+
+If you are upgrading from Earthly Satellites, the main benefit of using Earthly CI is that you no longer need to use a traditional CI in combination. This means less moving parts, simpler setup, slightly faster builds (no need to download Earthly during the build), and less bills to pay!
+
+## Getting started
+
+### 1. Register an account and create an org
+
+Follow the steps in the [Earthly Cloud overview](./overview.md#getting-started) to register an account and create an org.
+
+### 2. Gain access to Earthly CI
+
+Fill out the [Earthly CI signup form](https://forms.gle/iwSLazNv7FjYqyqY6). You will need to wait for an Earthly team member to approve your request.
+
+<!-- Use this after alpha.
+Fill out the [Earthly CI signup form](https://earthly.dev/signup/earthly-ci/). You will need to wait for an Earthly team member to approve your request.
+-->
+
+### 3. Ensure that you have the latest version of Earthly
+
+You need to make sure that you are using the [latest Earthly 0.7 release candidate](https://github.com/earthly/earthly/releases). Since this is a release candidate, it will not show up in the regular update channels, such as brew. You have to download the binary manually and add it to your `PATH`. Please note that there are some key changes from 0.6 to 0.7 - one of the important ones is that the cloud-based secret storage has a new format. See the [secrets migration guide](./cloud-secrets.md#migrating-from-the-old-0.6-experimental-version-of-earthly-secrets) for more information.
+
+<!-- Use this after alpha.
+Because this feature is under heavy development right now, it is very important that you use the latest version of Earthly available.
+
+**On Linux**, simply repeat the [installation steps](https://earthly.dev/get-earthly) to upgrade.
+
+**On Mac**, you can perform:
+
+```bash
+brew update
+brew upgrade earthly/earthly/earthly
+```
+-->
+
+### 4. Open the Earthly Web UI
+
+You can open the Earthly Web UI by running:
+
+```bash
+earthly web
+```
+
+This will associate your Earthly account with your GitHub login and then take you to the Earthly Web UI.
+
+### 5. Create your first Earthly pipeline
+
+In the Earthly Web UI, under the Organizations drop-down in the top-left of the screen, select the organization that was granted access to Earthly CI. Then follow the instructions on the screen to create a new Earthly CI project, add your first repository, create your pipeline via a new or an existing Earthfile, and then run your first build.
+
+For your very first CI pipeline you can use the following example:
+
+```Earthfile
+VERSION 0.7
+PROJECT my-org/my-project
```

Comment:
should there be a step to say "create a project" using `earthly project --org my-org create my-project`; otherwise users will get some warnings saying it doesn't exist.

Reply:
The UI flow goes through that. No need to create it in advance